app/views/casework/application_searches/_search_results_table_body.html.erb

```diff
@@ -7,8 +7,13 @@
   <td class="govuk-table__cell">
     <%= result.reference %>
   </td>
+  <% if FeatureFlags.post_submission_evidence.enabled? %>
+    <td class="govuk-table__cell">
+      <%= result.application_type.humanize %>
+    </td>
+  <% end %>
   <td class="govuk-table__cell">
-    <%= result.case_type.humanize %>
+    <%= result.case_type&.humanize  %>
```

Comment:
In the search table, case type is displayed but this is not present for pse applications, so have left blank for now following the existing behaviour for other columns